src/components/Button/src/Button.vue

```diff
@@ -189,24 +290,60 @@ export default {
 	computed: {
 		...resolveThemeableProps('button', [
 			'color',
+			'colorHover',
 			'size',
 			'textColor',
+			'textColorHover',
 			'variant',
 			'shape',
+			'shapeHover',
 			'align',
 			'fullWidth',
 			'pattern',
+			'borderWidth',
+			'borderWidthHover',
+			'borderColor',
+			'borderColorHover',
+			'boxShadow',
+			'boxShadowHover',
 		]),
 		style() {
 			const tokens = {
 				color: this.resolvedColor,
+				colorHover: this.resolvedColorHover,
 				textColor: this.resolvedTextColor,
+				textColorHover: this.resolvedTextColorHover,
+				shape: this.resolvedShape,
+				shapeHover: this.resolvedShapeHover,
+				borderWidth: this.resolvedBorderWidth,
+				borderWidthHover: this.resolvedBorderWidthHover,
+				borderColor: this.resolvedBorderColor,
+				borderColorHover: this.resolvedBorderColorHover,
+				boxShadow: this.resolvedBoxShadow,
+				boxShadowHover: this.resolvedBoxShadowHover,
+			};
+			return {
+				...setColorVariables(tokens, this.resolvedVariant),
 			};
-			return setColorVariables(tokens, this.resolvedVariant);
 		},
 		isDisabled() {
 			return this.disabled || this.loading;
 		},
+		fontSize() {
+			switch (this.resolvedSize) {
+			case 'small':
+				return '12px';
+			case 'medium':
+				return '14px';
+			case 'large':
+				return '16px';
+			default:
+				return undefined;
+			}
+		},
+		fontWeight() {
+			return this.resolvedSize === 'expandable' ? undefined : '500';
```

Comment:
i don't understand the logic behind this... why would whether or not a button is `expandable` change it's font-weight? 🤔